nn_adapt/solving.py

```diff
@@ -123,6 +123,7 @@ def get_solutions(
         solve(dFdq_transpose == dJdq, q_star, solver_parameters=sp)
         out["adjoint"] = q_star
     out["times"]["adjoint"] += perf_counter()
+
```

Comment:
It is good practice to avoid changing parts of the code that you aren't actually making changes to.